biodatasets/pharmaconer/pharmaconer.py

```diff
@@ -0,0 +1,253 @@
+                                {
+                                    "role": datasets.Value("string"),
+                                    "ref_id": datasets.Value("string"),
+                                }
+                            ),
+                        }
+                    ],
+                    "relations": [  # R line in brat
+                        {
+                            "id": datasets.Value("string"),
+                            "head": {
+                                "ref_id": datasets.Value("string"),
+                                "role": datasets.Value("string"),
+                            },
+                            "tail": {
+                                "ref_id": datasets.Value("string"),
+                                "role": datasets.Value("string"),
+                            },
+                            "type": datasets.Value("string"),
+                        }
+                    ],
+                    "equivalences": [  # Equiv line in brat
+                        {
+                            "id": datasets.Value("string"),
+                            "ref_ids": datasets.Sequence(datasets.Value("string")),
+                        }
+                    ],
+                    "attributes": [  # M or A lines in brat
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "ref_id": datasets.Value("string"),
+                            "value": datasets.Value("string"),
+                        }
+                    ],
+                    "normalizations": [  # N lines in brat
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "ref_id": datasets.Value("string"),
+                            "resource_name": datasets.Value("string"),
+                            "cuid": datasets.Value("string"),
+                            "text": datasets.Value("string"),
+                        }
+                    ],
+                },
+            )
+
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+        """
+        Downloads/extracts the data to generate the train, validation and test splits.
+
+        Each split is created by instantiating a `datasets.SplitGenerator`, which will
+        call `this._generate_examples` with the keyword arguments in `gen_kwargs`.
+        """
+
+        data_dir = dl_manager.download_and_extract(_URLS["pharmaconer"])
+
+        return [
```

Comment:
Here you should add the data for `subtrack 2`, which contains normalization (concept indexing ) information. See [here](https://temu.bsc.es/pharmaconer/) for details